### PR TITLE
Update rooms.json

### PR DESCRIPTION
**Pull request by LEDFlashing**
10-17 room setup

### CODE REVIEW

File: app/config/tstvhq/rooms.json [hunk 1]
```diff
@@ -184,8 +185,9 @@
             "id": "backParlor",
             "name": "Back Parlor",
             "type": "IFRAME",
-            "capacity": 30,
+            "capacity": 100,
             "muteRoom": true,
+            "hideVideo" true,
```
Comment (hndrewaall) [inline app/config/tstvhq/rooms.json:190]: this is missing a colon

File: app/config/tstvhq/rooms.json [hunk 2]
```diff
@@ -211,7 +213,9 @@
             "id": "ritualChamber",
             "name": "Ritual Chamber",
             "type": "IFRAME",
-            "capacity": 20,
+            "capacity": 100,
+            "muteRoom": true,
+            "hideVideo" true,
```
Comment (hndrewaall) [inline app/config/tstvhq/rooms.json:218]: also missing colon

File: app/config/tstvhq/rooms.json [hunk 3]
```diff
@@ -375,12 +383,13 @@
             "id": "tstHQ",
             "name": "Infernal Theater",
             "type": "IFRAME",
-            "capacity": 20,
+            "capacity": 100,
             "muteRoom": true,
+            "hideVideo" true,
```
Comment (hndrewaall) [inline app/config/tstvhq/rooms.json:388]: missing colon

File: app/config/tstvhq/rooms.json [hunk 4]
```diff
@@ -330,11 +334,15 @@
...
-            "type": "JITSI",
-            "capacity": 20,
-            "muteRoom": false,
+            "type": "IFRAME",
+            "capacity": 100,
+            "muteRoom": true,
+            "hideVideo": true,
             "description": "An intimate setting for covert conversations about the Satanic Agenda",
             "backgroundImage": "./https://tstvhq.s3.amazonaws.com/backgrounds/greenthe-revolt-of-the-angels002-e1552149546844.jpg",
+             "iframeOptions": {
```
Comment (hndrewaall) [inline app/config/tstvhq/rooms.json:343]: minor nitpick, but this is over-indented by one space

**Review by hndrewaall — CHANGES_REQUESTED** [review]

**Comment by hndrewaall** [comment]
Copied and fixed in #221 due to time concerns

File: app/config/tstvhq/rooms.json (hunk 1 above)
Comment (LEDFlashing) [reply app/config/tstvhq/rooms.json]: fixed

**Comment by hndrewaall** [comment]
Sorry @LEDFlashing I didn't see your commits! but works out fine either way. thanks for your help!

**Pull request closed without merging**